web/src/js/__tests__/ducks/tutils.js

```diff
@@ -77,6 +77,13 @@ export function TStore(){
                 type: "str",
                 value: "b"
             },
+            columns: {
+                choices: null,
+                default: ["Path"],
+                help: "Display specified columns",
+                type: "sequence of str",
+                value: ["", "Path", "Method", "Status", "Size", "Time"]
```

Comment:
The empty string for the TLS indicator is really awful here. Can we please use some other identifer? How about `column.name.slice(0, -6).toLowerCase()`? Alternative, we can add an explicit one.